README.md

```diff
@@ -199,20 +211,74 @@ _Note: This will be different if you installed DVWA into a different directory._
+#### I want to access the logs
 
-- - -
+If you are using Docker Desktop, logs can be accessed from the graphical application.
+Some minor details may change with newer versions, but the access method should be the same.
 
-## Troubleshooting
+![Overview of DVWA compose](./docs/graphics/docker/overview.png)
+![Viewing DVWA logs](docs/graphics/docker/detail.png)
 
-These assume you are on a Debian based distro, such as Debian, Ubuntu and Kali. For other distros, follow along, but update the command where appropriate.
+Logs can also be accessed from the terminal.
+
+1. Open a terminal and change its working directory to DVWA
+2. Show the merged logs
+
+    ```shell
+    docker compose logs
+    ```
+
+   In case you want to export the logs to a file, e.g. `dvwa.log`
+
+   ```shell
+   docker compose logs >dvwa.log
+   ```
+
+#### Port 80 is busy because I am already running something on it
+
+To workaround this, you can expose DVWA on a different port by changing the port binding in the `compose.yml` file.
+For example
+
+```yml
+ports:
+  - 8806:80
+```
+
+DVWA is now accessible at the newly configured port on your computer. For example, with the configuration like above, DVWA is available at `http://localhost:8806`.
+
+#### I am using a rootless container engine (e.g. Podman) and I cannot use privileged port 80
+
+If you are using a rootless container engine like [Podman](https://podman.io/), you will get `bind: permissions denied` error because ports lower than 1024 are privileged and requires root permissions.
+
+To workaround this, you can either
+
+- Bind DVWA's port to a port greater than or equal to 1024 by editing the `compose.yml` file.
+    For example
+
+    ```yml
+    ports:
+      - 8806:80
+    ```
+
+    DVWA is now accessible at the newly configured port on your computer. For example, with the configuration like above, DVWA is available at `http://localhost:8806`.
+- Or lower the start of unprivileged port by adding the following line to `/etc/sysctl.conf` (or any file in `/etc/sysctl.d/`)
+
+    ```text
+    net.ipv4.ip_unprivileged_port_start=80
+    ```
+
+#### Auto start DVWA when Docker runs
+
+The included [`compose.yml`](./compose.yml) file does not include the `restart` policy, thus DVWA will not restart when you restart Docker or reboot your system, according to the [Compose Specification](https://github.com/compose-spec/compose-spec/blob/master/spec.md#restart).
+Moreover, the [Compose Specification](https://github.com/compose-spec/compose-spec/blob/master/spec.md#restart) does not define a policy to auto start an entire Compose.
+
+This means every time you restart Docker or reboot your system, you need to open a terminal, change its working directory to DVWA and run `docker compose up -d`.
```

Comment:
The title suggests you are going to say how to auto start it but the text then says it won't auto start. Why not give instructions on how to may it auto start so people can do it if they want to.

Reply:
The current implementation cannot auto start due to `Cannot join network of a non-running container` error, so I have to change to a different network configuration. This requires a different `db_server`, and I decided to use `getenv()` for this field.
I did consider creating a separate config file for Docker, but this means more work if the config gets updated.